### PR TITLE
Swap 0bin for zerobin, re-order section

### PR DESCRIPTION
**Pull request by kewde**
### Description
+ Replace 0bin with zerobin (see vulnerabilities #454)
+ Move privacybin to first place (official zerobin successor + the maker of most private pastebins)

### HTML Preview

http://htmlpreview.github.io/?https://github.com/privacytoolsIO/privacytools.io/blob/zerobin/index.html

### CODE REVIEW

**Comment by Shifterovich** [comment]
Can't we get a better pic for zerobin?

**Comment by Shifterovich** [comment]
Apparently not. Though the white bottom is quite ugly.

**Comment by kewde** [comment]
How do you add a website without a logo. That's my attempt.

**Comment by infosec-handbook** [comment]
We should also add a warning for hastebin.com (in the "Worth Mentioning" section), because most web server security features are disabled and it runs external JS without any integrity checks.

**Comment by kewde** [comment]
@infosec-handbook 
External JS is bad mkay.

**Comment by kewde** [comment]
@Shifterovich 
Ghostbin: keep or remove?
https://github.com/privacytoolsIO/privacytools.io/issues/454#issuecomment-385314214

**Comment by Shifterovich** [comment]
Keep, but:

> Regarding Ghostbin, we should warn users that while Ghostbin - the software - is good, ghostbin.com's security is worrisome.

File: index.html [hunk 1]
```diff
@@ -2593,36 +2593,39 @@ <h1><a href="#paste" class="titleanchor"><span class="glyphicon glyphicon-link">
...
         </div><!-- /.col-sm-4 -->
         <div class="col-sm-4">
           <div class="panel panel-info">
             <div class="panel-heading">
-              <h3 class="panel-title">PrivateBin</h3>
+              <h3 class="panel-title">Zerobin</h3>
             </div>
             <div class="panel-body">
-              <p><img src="img/tools/PrivateBin.png" alt="PrivateBin" align="right" style="margin-left:5px;">PrivateBin is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted in the browser using 256bit AES.</p>
+              <p><img src="img/tools/ZeroBin.png" alt="ZeroBin" align="right" style="margin-left:5px;">Zerobin is a minimalist, open source online pastebin where the server has zero knowledge of pasted data. Data is encrypted/decrypted in the browser using 256bit AES.</p>
```
Comment (EvasivePangolin) [inline index.html]: Change Zerobin to ZeroBin.net to differentiate between the website and the [outdated project](https://github.com/sebsauvage/ZeroBin).

**Comment by kewde** [comment]
@Shifterovich

**Comment by Shifterovich** [comment]
maybe explain in short why the website's security is worrisome?

**Comment by kewde** [comment]
I'll add a hyperlink to #454 - the issue with all the details.

**Pull request merged**